<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CHILL HOME USERS</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="editName">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddVehicle">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>610</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>차량 정보</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUserAdd">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>610</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>등록</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>701</width>
     <height>441</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>index</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>User ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Birthday</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>tel</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>level</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>createDate</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>131</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string>tab</string>
   </property>
   <widget class="QPushButton" name="tabMain">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>89</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>MAIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tabLog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>89</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>LOG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tabUsers">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>89</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>USER</string>
    </property>
   </widget>
  </widget>
  <widget class="QScrollBar" name="verticalScrollBar">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>160</y>
     <width>20</width>
     <height>401</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUserUpdate">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>610</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>수정</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUserDelete">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>610</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>삭제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
